app/mailers/application_mailer.rb

```diff
@@ -1,7 +1,10 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: "DEV Community <#{ApplicationConfig['DEFAULT_SITE_EMAIL']}>"
   layout "mailer"
-  default template_path: ->(mailer) { "mailers/#{mailer.class.name.underscore}" }
+
+  default(
+    from: -> { "DEV Community <#{SiteConfig.default_site_email}>" },
```

Comment:
Good idea, but I would do it in a following PR :)

```shell
➜  devto git:(rhymes/add-wait-for-javascript-system-tests) rg "DEV Community" | wc -l
      40
```

The word `DEV` itself appears 486 times :D

Reply:
I think the goal of this PR is to remove the last known environment variable we don't need, then merge https://github.com/thepracticaldev/dev.to/pull/5400 - after that we can continue with our generalizations efforts. We have a beta tester that has a lot of insights in what we could prioritize

Reply:
This is probably a question we'll encounter a lot of in the generalization process.

Since the other `from`s are `DEV Digest ...` and `DEV Pro Memberships ...` (which I promise we'll eventually ship!) I'd say we can make all of these into `"#{SiteConfig.community_name} Community <#{SiteConfig.default_site_email}>"`

Currently `ENV["COMMUNITY_NAME"]` is "DEV" and then depending on context it could be DEV Community or just DEV where used. Every user of this app will be a "community".

Then there's the question of "should the whole from: be customizable at this granularity" and that is a bridge we should cross later, but all else equal we should probably try to re-use different config options where it makes sense rather than get spread too thin.

Anyway, agree this is for another PR.